examples/summarization/bart/README.md

```diff
@@ -1,4 +1,7 @@
-### Get Preprocessed CNN Data
+### Get CNN Data
```

Comment:
can we delete the bart subdir and just have `examples/summarization`

Reply:
Are you suggesting to  merge bertabs and bart subdir into one?

Reply:
I'm suggesting moving the combined contents of the t5/ and bart/ subdirs into the parent directory, so that the tree looks 
```bash
examples/summarization:
- evaluate_cnn.py
- finetune.py
- finetune_bart.sh, finetune_tiny.sh, finetune_t5.sh (old run_train.sh)
- test_summarization_examples.py (old test_bart_examples.py)
- utils.py 
- README.md
examples/summarization/bertabs/  #same as before
```
This list not exhaustive, there might be more files). Sorry for not being clear.

Reply:
Thanks for the detailed info :)